joint_teapot/utils/joj3.py

```diff
@@ -0,0 +1,104 @@
+import csv
+import json
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+from joint_teapot.utils.logger import logger
+
+
+def generate_scoreboard(score_file_path: str, scoreboard_file_path: str) -> None:
+    if not score_file_path.endswith(".json"):
```

Comment:
We can just validate the path in cli, or just remove it in case we need to write to some special files.